introduction.md

```diff
@@ -1,3 +1,15 @@
 Hello, my name is Kevin, and I'm a web developer.
 
 My favorite Code School path is the JavaScript path.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?